packages/epics/src/governance/components/document-card.tsx

```diff
@@ -0,0 +1,88 @@
+import { type Person, type Document } from '@hypha-platform/core';
```

Comment:
Please declare the required interface on the UI component. No tight coupling to the data model please